### PR TITLE
mask generation for text locatiozation

### PR DESCRIPTION
**Pull request by aarora8**
Adding data download
First version of mask generation.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +57,6 @@ GSYMS
 /egs/*/v*/plp
 /egs/*/v*/exp
 /egs/*/v*/data
+
+# .idea
+.idea/
```
Comment (hhadian) [inline .gitignore]: Is this for some kind of IDE?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # waldo
- image-segmentation and text-localization 
+ image-segmentation and text-localization
+
+
+/Users/ashisharora/Desktop/scale2018/waldo_text_loc_mask/waldo
```
Comment (hhadian) [inline README.md]: I guess you have added this by mistake.

File: egs/madcat_arabic/v1/path.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+export LC_ALL=C
```
Comment (hhadian) [inline egs/madcat_arabic/v1/path.sh]: add a new line at the end. also for other files.

**Comment by danpovey** [comment]
I'm fine to merge without too much review for now.

File: .gitignore (hunk 1 above)
Comment (aarora8) [reply .gitignore]: sorry, Yes i was working locally. I added it to my local copy. I will remove it

File: README.md (hunk 2 above)
Comment (aarora8) [reply README.md]: I will remove it also

File: egs/madcat_arabic/v1/path.sh (hunk 3 above)
Comment (aarora8) [reply egs/madcat_arabic/v1/path.sh]: I will make these change quickly.

File: egs/madcat_arabic/v1/run.sh [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                           --download_dir3 $download_dir3 data/local/$dataset
+  done
+fi
+
+mkdir -p data/{train,test,dev}/masks
+if [ $stage -le 1 ]; then
+  echo "$0: Extracting mask from page image for dataset:  $dataset. "
+  echo "Date: $(date)."
+  for dataset in test dev train; do
+  dataset_file=/home/kduh/proj/scale2018/data/madcat_datasplit/ar-en/madcat.$dataset.raw.lineid
```
Comment (hhadian) [inline egs/madcat_arabic/v1/run.sh]: Fix indentation here. 
Please remove the [WIP] tag when the PR is ready for merge.

**Comment by aarora8** [comment]
Thank you. It is ready for merge

**Comment by hhadian** [comment]
Thanks Ashish! Merged.

**Pull request merged**